doc/src/sgml/meson.build

```diff
@@ -81,19 +81,29 @@ postgres_sgml_valid = custom_target('postgres.sgml.valid',
 # Full documentation as html, text
 #
 if xsltproc.found()
+  xsltproc_wrapper = find_program('xsltproc_depfile_wrapper.py')
   xsltproc_flags = [
     '--stringparam', 'pg.version', pg_version,
     '--param', 'website.stylesheet', '1'
   ]
 
+  xsltproc_cmd = [
+    xsltproc_wrapper,
+    '--xsltproc', xsltproc,
+    '--targetname', '@OUTPUT@',
+    '--depfile', '@DEPFILE@',
+    '--',
+      '--path', '@OUTDIR@',
+  ]
 
   # FIXME: Should use a wrapper around xsltproc --load-trace to compute a
   # depfile
   html = custom_target('html',
     input: ['stylesheet.xsl', 'postgres.sgml'],
     output: 'html',
+    depfile: 'html.dep',
```

Comment:
the dependency targets are typically called something.d, maybe stick with that here?